lib/auth-providers/cmd.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Fetches a bearer token via comamnd
+ */
+
+'use strict';
+
+// for API compatability
+/* eslint no-sync: 0 */
+const spawnSync = require('child_process').spawnSync;
+
+module.exports = {
+  refresh: function (config) {
+    return new Promise((resolv, reject) => {
```

Comment:
is there any reason why it is not `resolve` but `resolve`?

Reply:
I've always been used to that style... then again haven't used non `async/await` in a longggg time. If this is really necessary I'll change it.